docs/guides/key-manager/upgrade-lsp6.md

```diff
@@ -0,0 +1,246 @@
+---
+sidebar_label: 'Upgrade LSP6 Key Manager'
+sidebar_position: 4
+---
```

Comment:
Maybe you can add a page description?

https://docusaurus.io/docs/markdown-features/head-metadata#markdown-page-description

As the first full line here may not be super relevant.

Keep it under 160 chars.

This can also be applied to other pages to improve SEO.